src/mscorlib/src/System/Security/SecurityException.cs

```diff
@@ -663,5 +663,21 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue(UrlName, m_url, typeof(String));
 #endif // FEATURE_CAS_POLICY            
         }
+
+#if !FEATURE_CAS_POLICY
+            // Stubs for surface area compatibility only
+            public SecurityException(string message, System.Type type) { }
+            public SecurityException(string message, System.Type type, string state) { }
+            public object Demanded { get { throw null; } set { } }
+            public object DenySetInstance { get { throw null; } set { } }
+            public System.Reflection.AssemblyName FailedAssemblyInfo { get { throw null; } set { } }
+            public string GrantedSet { get { throw null; } set { } }
+            public System.Reflection.MethodInfo Method { get { throw null; } set { } }
+            public string PermissionState { get { throw null; } set { } }
+            public System.Type PermissionType { get { throw null; } set { } }
+            public object PermitOnlySetInstance { get { throw null; } set { } }
```

Comment:
It would be about as hard to just make them work ... all of them can be just trivial getters/setters that don't do anything interesting.